rafthttp/stream.go

```diff
@@ -38,6 +39,16 @@ const (
 	streamBufSize = 4096
 )
 
+var (
+	errUnsupportedStreamType = fmt.Errorf("unsupported stream type")
+
+	streamSupportMap = map[streamType]*semver.Version{
```

Comment:
what is a max version map? I do not want to have an ad hoc solution just for this pr.